sorc/ncep_post.fd/TTBLEX.f

```diff
@@ -112,5 +112,132 @@ SUBROUTINE TTBLEX(TREF,TTBL,ITB,JTB,KARR,PMIDL         &
       ENDDO
 !
       RETURN
-      END
+      END SUBROUTINE TTBLEX
 !&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&
+!
+!-------------------------------------------------------------------------------------
+!
```

Comment:
As what I comment in MISCLN.f for scheme 2,
 "_After the testing in HWT SFE 2021, it seems that Scheme 1 had been working well and SPC was satisfied with it. So to make the code management and maintenance simple and easy, I suggest to set Scheme 1 as the only scheme for Effective-Layer, and remove the code related to Scheme 2 (in this MISCLN.f, and in UPP_PHYSICS.f and in TTBLEX.f). Just my suggestion. These code still can be kept._"
This subroutine TTBLEX1D is used only for scheme 2 in MISCLN.f for searching for the Effective-Layer. It could be removed for code clean if scheme 2 is not necessary.